cl/corpus_importer/tasks.py

```diff
@@ -329,20 +329,24 @@ def get_and_save_free_document_report(
                 "to failure to get Nonce."
             )
         elif isinstance(exc, (RequestException, ReadTimeoutError)):
-            msg = (
-                "Unable to get free document report results at %s (%s to %s)."
-            )
+            msg = "Unable to get free document report results"
         elif isinstance(exc, PacerLoginException):
-            msg = (
-                "PacerLoginException while getting free docs at %s (%s to %s)."
-            )
+            msg = "PacerLoginException while getting free docs"
         elif isinstance(exc, ParsingException):
-            msg = "Didn't get nonce at %s (%s to %s)."
+            if "nonce" in exc.message:
+                msg = "Didn't get Nonce"
+            elif "XML" in exc.message:
+                msg = (
+                    "Written opinion reports are blocked. Please "
+                    "contact the court director"
+                )
```

Comment:
Did you test `exc.message`?

Reply:
Fixed the exception description evaluation.